gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/work/DatasetStats.java

```diff
@@ -22,13 +22,17 @@
 import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
 
+
+/**
+ * Stats for a dataset that was committed.
```

Comment:
"committed" or merely "processed"?  e.g. is it not possible for `successfullyCommitted == false` and `numCommittedWorkunits == 0`?

also, looks like a stray blank line in the javadoc comment

Reply:
Ah good point, I was thinking about some other partial commit semantic but it doesn't apply here, it would still be considered successfully committed in the context of these stats